src/pages/NotFoundView/NotFoundView.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {Link} from 'react-router';
+
+export class NotFoundView extends React.Component {
```

Comment:
Why does this class have the suffix `View`, when other classes in the `pages` directory have suffix `Page`?